packages/subgraph/test/helpers/initializers.ts

```diff
@@ -81,6 +81,7 @@ export const getOrInitFlowOperator = (
             updatedAtBlockNumber,
             permissions: 0,
             flowRateAllowanceGranted: "0",
+            allowance: "0",
```

Comment:
please remove this, we ideally want to remove code from `packages/subgraph/test`